solver/interactiveSolver.py

```diff
@@ -1024,13 +1048,18 @@ def importDump(self, shmName):
             "samus": '4',
             "items": '5',
             "boss": '6',
-            "events": '7'
+            "events": '7',
+            "inventory": '8',
         }
 
         currentState = dumpData["currentState"]
         self.locDelta = 0
 
-        for dataType, offset in dumpData["stateDataOffsets"].items():
```

Comment:
why are you doing a manual sort ?
we use python >= 3.6 which keep the order of insertion of keys in dictionaries.